components/style/core/base.less

```diff
@@ -61,6 +61,10 @@ a {
   cursor: pointer;
   transition: color .3s ease;
 
+  &:focus {
+    text-decoration: underline;
```

Comment:
@afc163 Could you please direct me to where in the specification `text-decoration: underline` is prohibited? 

And if underline is prohibited, do you have another suggestion for visually indicating keyboard focus on link elements?